src/app/rpmostree-builtin-kargs.c

```diff
@@ -84,18 +186,16 @@ rpmostree_ex_builtin_kargs (int            argc,
                                        error))
     return EXIT_FAILURE;
 
-  /* TODO: uncomment these lines when finish implementing editor feature
-   *
-   * if (opt_editor && (opt_import_proc || opt_kernel_delete_strings ||
-   *   opt_kernel_replace_strings ||  opt_kernel_append_strings))
-   *  {
-   *    We want editor command to achieve all the other functionalities
-   *   Thus erroring out ahead of time when multiple options exist
-   *   g_set_error (error, G_IO_ERROR, G_IO_ERROR_INVALID_ARGUMENT,
-   *                "Cannot specify --editor with other options");
-   *   return EXIT_FAILURE;
-   * }
-   */
+   if (opt_editor && (opt_kernel_delete_strings ||
+       opt_kernel_replace_strings ||  opt_kernel_append_strings))
+     {
+      /* We want editor command to achieve all these functionalities
+       * Thus erroring out ahead of time when these strings exist
+       */
+      g_set_error (error, G_IO_ERROR, G_IO_ERROR_INVALID_ARGUMENT,
+                   "Editor command is already enough to modify the kernel arguments");
```

Comment:
Let's be more explicit here, e.g. `"Cannot specify --editor with --replace, --delete, or --append"` ?

Reply:
Sure =)